recipes/plugins_development.ipynb

```diff
@@ -172,56 +218,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "project_id = \"<PROJECT ID>\""
+    "# Here replace 'plugin_folder' with the actual name of the folder\n",
+    "from plugin_folder.main import PluginHandler"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
-   "metadata": {
-    "tags": [
-     "skip"
-    ]
-   },
+   "metadata": {},
    "source": [
-    "## Setup a project"
+    "#### 2.2 Absolute path import"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "tags": [
-     "skip"
-    ]
-   },
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "from pathlib import Path\n",
+    "\n",
+    "# Input the path to the plugin folder (it should include the folder), for example '/path/to/plugin_folder'\n",
+    "plugin_path = \"<INSERT PATH TO PLUGIN FOLDER>\"\n",
+    "\n",
+    "module_path = str(Path(plugin_path).parent.absolute())\n",
+    "\n",
+    "# We are inserting the path in the system PATH to be able to import the module in the next line\n",
+    "sys.path.insert(0, module_path)\n",
+    "\n",
+    "# In the next line replace 'plugin_folder' with the actual name of the folder\n",
+    "from plugin_folder.main import PluginHandler"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Testing the plugin locally\n",
+    "\n",
+    "In this we will show you how to test your plugin locally before uploading it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# create project\n",
-    "json_interface_object_detection = {\n",
-    "    \"jobs\": {\n",
-    "        \"JOB_0\": {\n",
-    "            \"content\": {\n",
-    "                \"categories\": {\n",
-    "                    \"OBJECT_A\": {\"children\": [], \"name\": \"Object A\", \"color\": \"#733AFB\"},\n",
-    "                    \"OBJECT_B\": {\"children\": [], \"name\": \"Object B\", \"color\": \"#3CD876\"},\n",
-    "                },\n",
-    "                \"input\": \"radio\",\n",
-    "            },\n",
-    "            \"instruction\": \"Categories\",\n",
-    "            \"isChild\": False,\n",
-    "            \"tools\": [\"rectangle\"],\n",
-    "            \"mlTask\": \"OBJECT_DETECTION\",\n",
-    "            \"models\": {},\n",
-    "            \"isVisible\": True,\n",
-    "            \"required\": 1,\n",
-    "        }\n",
-    "    }\n",
-    "}\n",
-    "project_id = kili.create_project(\n",
-    "    input_type=\"IMAGE\",\n",
-    "    json_interface=json_interface_object_detection,\n",
-    "    title=\"export kili project tutorial\",\n",
-    ")[\"id\"]"
```

Comment:
@gheorghetutunaru 

All notebooks in the SDK doc are tested: https://github.com/kili-technology/kili-python-sdk/blob/master/tests/test_notebooks.py

This PR removes all the code related to project creation, and breaks the CI since the notebook can't run end-to-end anymore: https://github.com/kili-technology/kili-python-sdk/actions/runs/4045142408/jobs/6956217482

If you want to hide some cells during the notebook -> markdown conversion, you can tag a cell with those keywords: https://github.com/kili-technology/kili-python-sdk/blob/a70cb0f6cfd75c9f5aa2dc4ad83b98e0c7020d90/docs/utils.py#L116 and it won't show up in the markdown.

Make sure to run the notebook tests when you modify a notebook, thanks